src/webgpu/api/operation/device/lost.spec.ts

```diff
@@ -13,7 +14,14 @@ class DeviceLostTests extends Fixture {
   readonly kDeviceLostTimeoutMS = 2000;
 
   getDeviceLostWithTimeout(lost: Promise<GPUDeviceLostInfo>): Promise<GPUDeviceLostInfo> {
-    return raceWithRejectOnTimeout(lost, this.kDeviceLostTimeoutMS, 'device was not lost');
+    // Set `noRaceWithRejectOnTimeout` to false, and then restore it after calling
+    // `raceWithRejectOnTimeout`. Here, it is used to ensure a Promise is not resolved, not
+    // to ensure a Promise must settle by some deadline.
+    const orig = globalTestConfig.noRaceWithRejectOnTimeout;
+    globalTestConfig.noRaceWithRejectOnTimeout = false;
```

Comment:
If we go with this approach I'd prefer to just split out a version of raceWithRejectOnTimeout which doesn't check the globalTestConfig.